Assets/XXHash.cs

```diff
@@ -0,0 +1,333 @@
+/*
```

Comment:
@ST-DDT Can you please check if we are allowed to copy this file into our project this way?

Reply:
AFAICT its OK to do it like this.